Fastfile

```diff
@@ -140,10 +134,34 @@ def _build_with_gym()
   end
 end
 
+def _get_export_method()
+  export_method = "enterprise"  # Default export method is `enterprise` since this is the most commonly used
+  if ENV['GYM_EXPORT_OPTIONS'] != nil
+    export_method = get_info_plist_value(path: ENV['GYM_EXPORT_OPTIONS'], key: "method")
+  else
+    UI.message("Falling back to enterprise `export_method` since `GYM_EXPORT_OPTIONS` is not defined")
+  end
+  export_method
```

Comment:
Is this actually required at the end of the function in ruby to return it?

Reply:
Yup. In Ruby people tend to write functions that are as short as possible, frequently just one line with at most one conditional branch. So this function would be 
```ruby
def _get_export_method()
  ENV['GYM_EXPORT_OPTIONS'] != nil ? get_info_plist_value(path: ENV['GYM_EXPORT_OPTIONS'], key: "method") : "enterprise"
end
```
but alas then you lose the ability to do side-effecty things like `UI.message`. So I think this is fine as is.

Reply:
What if you have a separate function `_fallback_to_enterprise_export_method()` that does the `UI.message` and returns the default?
That way we can keep the one liner here that @neilkimmett suggests.

Reply:
@NikolaKirev updated